test/integration/preflight/preflight_test.go

```diff
@@ -70,7 +103,7 @@ func (p PlaybookTest) checkExitCode(t *testing.T, got, want int, cmd *exec.Cmd,
 	if got == want {
 		return
 	}
-	t.Fatalf("got exit code %v, want %v", got, want)
+	t.Fatalf("%s got exit code %v, want %v", p.Path, got, want)
```

Comment:
The path + a copy-pasteable command should appear in the log from `logCmdAndOutput`, did you see that?